sig-scalability/configs-and-limits/thresholds.md

```diff
@@ -2,100 +2,73 @@
-| Number of pods per deployment       | TBD            |             | 10000          |
-| Number of jobs per ns               | TBD            |             | 1000           |
-| Number of daemon sets per ns        | TBD            |             | 100            |
-| Number of stateful sets per ns      | TBD            |             | 100            |
-| Number of secrets per ns            | TBD            |             | TBD            |
-| Number of secrets per pod           | TBD            |             | TBD            |
-| Number of config maps per ns        | TBD            |             | TBD            |
-| Number of config maps per pod       | TBD            |             | TBD            |
-| Number of storageclasses            | TBD            |             | TBD            |
-| Number of roles and rolebindings    | TBD            |             | TBD            |
+There are couple caveats to the thresholds we are presenting below:
+1. In majority of cases, thresholds are NOT hard limits - crossing
+   the limit results in degraded performance and doesn't mean cluster
+   immediately fails over.
+1. **Many of the tresholds (for cluster scope) are given for the largest
+   possible cluster. For smaller clusters, the limits are proportionally
+   lower.**
+1. The thresholds may differ (hopefully be non-decreasing) across Kubernetes
+   releases. The threshold below are given for Kubernetes head. <br/>
+   **TODO:** We are planning to start versioning the table below, but we
+   are not there yet.
+1. Given that configuration influences thresholds, we are assuming vanilla
+   Kubernetes setup.
 
-There are also thresholds for other types, but for those the numbers depend
-also on the environment (bare metal or which cloud provider) the cluster is
-running in. These include:
+The table below is **NOT exhaustive** - more content is coming soon.
 
-| Quantity                                  | Head threshold | 1.8 release | Long term goal |
-|-------------------------------------------|----------------|-------------|----------------|
-| Number of ingresses                       | TBD            |             | TBD            |
-| Number of PersistentVolumes               | TBD            |             | TBD            |
-| Number of PersistentVolumeClaims per ns   | TBD            |             | TBD            |
-| Number of PersistentVolumeClaims per node | TBD            |             | TBD            |
+| Quantity               | Threshold scope=namespace | Threshold: scope=cluster |
+|------------------------|---------------------------|--------------------------|
+| #Nodes                 | n/a                       | 5000                     |
+| #Namespaces            | n/a                       | 10000                    |
+| #Pods                  | 3000                      | 150000                   |
+| #Pods per node         | min(110, 10*#cores)       | min(110, 10*#cores)      |
```

Comment:
NIT: +1 to Shyam's comment, not sure how to understand the scope=namespace here. Having a value here suggests that it may differ in a different scope. 
What about putting n/a here, as we have in #nodes ?

Reply:
The reasons I didn't put here (but I put elsewhere, e.g. for nodes) is that things like nodes, namespaces, PVs are non-namespaced resources. Whereas pods are namespaced.
So theoretically, I can imagine reasons why this may differ for namespace and cluster scopes.

Reply:
Wojtek has a fair point. We might potentially limit `(pods per namespace) per node` differently than `pods per node` in future.